eng/CodeGeneration.targets

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <_AutoRestVersion>https://github.com/Azure/autorest/releases/download/autorest-3.0.6191/autorest-3.0.6191.tgz</_AutoRestVersion>
     <_AutoRestCoreVersion>3.0.6237</_AutoRestCoreVersion>
-    <_AutoRestCSharpVersion>https://github.com/Azure/autorest.csharp/releases/download/1.0.0-dev.20200311.4/autorest-csharp-v3-1.0.0-dev.20200311.4.tgz</_AutoRestCSharpVersion>
+    <_AutoRestCSharpVersion>https://github.com/Azure/autorest.csharp/releases/download/1.0.0-dev.20200312.2/autorest-csharp-v3-1.0.0-dev.20200312.2.tgz</_AutoRestCSharpVersion>
```

Comment:
Are we at a place were we want to have PR's automatically submitted for these updates?

Reply:
I would love it if something can update the version and run the generator on everything.

Reply:
I'm happy to help you put together a pipeline that can do this. Lets sync up at some point in the coming weeks to see how we can do this.